econml/cate_estimator.py

```diff
@@ -348,3 +408,94 @@ def coef__interval(self, *, alpha=0.1):
     @BaseCateEstimator._defer_to_inference
     def intercept__interval(self, *, alpha=0.1):
         pass
+
+
+class StatsModelsCateEstimatorDiscreteMixin(BaseCateEstimator):
+
+    def _get_inference_options(self):
```

Comment:
Could we ever have an estimator that supports both discrete and continuous statsmodels inference (depending on the treatments it's applied to)?  If so, is the fact that each mixin sets the `statsmodels` dictionary in a mutually exclusive way a problem?

Reply:
I would move it to a future PR. It might be problematic. Because for DML with discrete treatments you should use StatsModelsInference, but with DRLearner you should use StatsModelsInferenceDiscrete. So the user would need to know when to set the correspnding parameter of the merged class, which might be weird. Now the docstring points to the right class. We could in the back end have a parent class of both classes, but not change the front end of two classes.

Reply:
Ok.  No need to worry too much about such complications at this stage I guess.